copy/multiple.go

```diff
@@ -87,17 +89,43 @@ func (c *copier) copyMultipleImages(ctx context.Context, policyContext *signatur
 	}
 
 	// Copy each image, or just the ones we want to copy, in turn.
+	optionsInstances := options.Instances
 	instanceDigests := updatedList.Instances()
```

Comment:
`instanceDigests` is, AFAICS, not used inside the added code, so please set it just before the first user.

---

All of the new code computing the `optionsInstances` data structure from `options.*` seems like a good candidate for a separate function; it is clearly “functional”, unlike the more imperative surrounding code, and potentially easy to unit-test if split. (I don’t think a unit test is blocking this PR.)

Reply:
Split it off into a `determineSpecificImages()` function.